docs/containers/quickstart-node.md

```diff
@@ -0,0 +1,118 @@
+---
+Order: 7
+Area: containers
+TOCTitle: Node.js
+ContentId: A963901F-BF3F-455F-AD75-AB54EAE72BEF
+PageTitle: Build and run a Node.js app in a container
+DateApproved:
+MetaDescription: Develop, build, and debug a Node.js app in a Docker container, using Visual Studio Code.
+---
+# Build and run a Node.js app in a container
+
+In this guide you will learn how to:
+
+- Create a `Dockerfile` file for an [Express](https://expressjs.com/) Node.js service container
+- Build, run, and verify the functionality of the service
+- Debug the service running within a container
+
+## Prerequisites
+
+- Both Docker and the VS Code Docker extension must be installed as described on the [[Home page|Home#installation]]
```

Comment:
This link should be updated to the format and renamed files used in VS Code repo:
`[Overview](overview.md#installation)`

Reply:
Updated.